### ISSUE TITLE
DoMulti panic when used with no commands

### ISSUE THREAD

**Issue by zhiburt** [rueian/rueidis#82]
Hi @rueian, a small note.

```go
rdb.DoMulti(ctx)
```

```text
panic: runtime error: index out of range [0] with length 0

goroutine 1 [running]:
github.com/rueian/rueidis.(*pipe).DoMulti(0xc0000e8420, {0x7cf188, 0xc000092780}, {0xa24cc0?, 0x0, 0x0})
        /home/mzhiburt/go/pkg/mod/github.com/rueian/rueidis@v0.0.70/pipe.go:600 +0x934
github.com/rueian/rueidis.(*mux).pipelineMulti(0xc000092720, {0x7cf188, 0xc000092780}, {0xa24cc0, 0x0, 0x0})
        /home/mzhiburt/go/pkg/mod/github.com/rueian/rueidis@v0.0.70/mux.go:189 +0x9e
github.com/rueian/rueidis.(*mux).DoMulti(0x0?, {0x7cf188?, 0xc000092780?}, {0xa24cc0?, 0xc000094550?, 0x9e2b40?})
        /home/mzhiburt/go/pkg/mod/github.com/rueian/rueidis@v0.0.70/mux.go:151 +0x35
github.com/rueian/rueidis.(*singleClient).DoMulti(0xc0000b06f0, {0x7cf188, 0xc000092780}, {0xa24cc0?, 0x0, 0x0})
        /home/mzhiburt/go/pkg/mod/github.com/rueian/rueidis@v0.0.70/client.go:50 +0x9a
```

**Comment by rueian** [comment]
Hi @zhiburt, thank you for reporting this. This will be fixed in couple of days.

**Comment by rueian** [comment]
Hi @zhiburt, this is fixed in v0.0.71.

Please feel free to open a new issue if you find any problems. Thank you very much! 🤗
